api-auto-registration/usage.hbs.md

```diff
@@ -1,14 +1,30 @@
 # API Auto Registration Usage
 
-## <a id='usage'></a>Using App Accelerator Template
+On a high level, API Auto Registration requires users to perform two steps: 1. generate API spec 2. create APIDescriptor CR.
```

Comment:
> generate API spec 

There are two paths here. One is to generate and expose API spec from the application, the other one is generate and publish API spec to a static location. Not sure what 's the best way to word it in this section, but we should keep this consistent as what we discussed in the key-concept section.

> requires users to perform

This makes the two steps sound like an active actions rather than something they can get for free. Maybe consider changing to something like: `requires two prerequisites: 1. somewhere to retrieve API spec from; 2. APIDescriptor CR getting created in the run clusters`?

Reply:
Updated. Check it out

Reply:
Looks good! Thanks! (I don't see the option to resolve this conversation but you can consider this one resolved.)